.gitignore

```diff
@@ -3,6 +3,5 @@
 bower.json
 cc-test-reporter
 coverage/
-dist/
```

Comment:
Its not planned to host the bundled files in GH, so this change is not acceptable.

Reply:
added -dist/ to gitignore.

Reply:
<img width="553" alt="image" src="https://github.com/kurkle/chartjs-chart-sankey/assets/94789179/48fb691f-101c-46a3-95e8-39f780dc15a2">

PR request failed due to the above error